src/custom/state/application/hooks.ts

```diff
@@ -42,3 +48,31 @@ export function useAddPopup(): (content: PopupContent, key?: string, removeAfter
     [dispatch]
   )
 }
+
+// Allow show popup
+// actions to use followPendingTxPopup atom
+export const useFollowPendingTxPopup = () => {
+  const [, _setFollowPendingPopup] = useAtom(handleFollowPendingTxPopupAtom)
+  const [, _setHidePopup] = useAtom(handleHidePopupPermanentlyAtom)
```

Comment:
Use https://jotai.org/docs/utils/use-update-atom for when you only want the setter